src/components/TablePagination/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pagination } from 'react-rainbow-components';
+
+export default function TablePagination(props) {
+    const {
+        pages,
+        activePage,
+        onChange,
+    } = props;
+
+    if (pages > 1) {
+        return (
+            <div className="react-rainbow-admin-orders_pagination">
```

Comment:
this classname is currently used? if is used then change class name indicating the component like:
```react-rainbow-admin-table-pagination```